tests/Invoke-DbaDbTransfer.Tests.ps1

```diff
@@ -0,0 +1,127 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys
+        [object[]]$knownParameters = @(
+            'SqlInstance',
+            'SqlCredential',
+            'DestinationSqlInstance',
+            'DestinationSqlCredential',
+            'Database',
+            'DestinationDatabase',
+            'BatchSize',
+            'BulkCopyTimeOut',
+            'InputObject',
+            'EnableException',
+            'CopyAllObjects',
+            'CopyAll',
+            'SchemaOnly',
+            'DataOnly',
+            'ScriptingOption',
+            'ScriptOnly'
+        )
+        $knownParameters += [System.Management.Automation.PSCmdlet]::CommonParameters
+        It "Should only contain our specific parameters" {
+            $knownParameters | Where-Object { $_ } | Should -BeIn $params
+            $params | Should -BeIn ($knownParameters | Where-Object { $_ })
```

Comment:
I'd prefer we use the same pattern as our other tests

Reply:
https://github.com/sqlcollaborative/dbatools/blob/fc1bb047ad6357f4366e8e9e4a92448923125209/tests/Copy-DbaAgentJob.Tests.ps1#L11

Reply:
While I understand the concern and will address it, the pattern used in the tests doesn't show which parameter is missing and I find it inconvenient.

Reply:
Understood, a new issue can be created to address that but at this time this is the pattern we are using.

Reply:
All of our test will be touched later this year anyway and many are going to get rewritten and restructured....but for now we need to keep the same pattern going so when that times comes it can be a mass find/replace for the new pattern we chose to use.